src/wayland/output/handlers.rs

```diff
@@ -19,9 +19,10 @@ where
     D: GlobalDispatch<WlOutput, WlOutputData>,
     D: Dispatch<WlOutput, OutputUserData>,
     D: 'static,
+    D: OutputHandler,
```

Comment:
some small style nit, I would prefer if we can keep the `'static` lifetime last

Reply:
Done